README.md

```diff
@@ -198,6 +199,14 @@ x-transportd:
     allowedaudience: ""
     # ([]string) Acceptable issuer strings.
     allowedissuers:
+  validateheaders:
+    # (map[string][]string) allowed list of headers whose values to check
+    allowed:
+      accept:
+        - "text/json"
+        - "text/html"
+    # (string) the delimiter to use for splitting header values when they come in single line
+    split: ","
```

Comment:
task: update this example in the readme, either remove it or use a simple example like what's in the `/docs` directory

Reply:
good call, will fix it now